jest.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+    plugins: ["jest"],
+    extends: ["plugin:jest/recommended"],
+    rules: {
+        "jest/no-standalone-expect": [
+            "error",
+            {
+                additionalTestBlockFunctions: [
+                    // Tested and assertions in `before`/`beforeEach` blocks will fail
+                    // the test as expected
+                    "before",
```

Comment:
I am not aware of a `before` in the Jest API.
But I would change this to `beforeAll`.

```suggestion
                    "beforeAll",
```

Reply:
👍 Probably just typo/assumptions from Mocha. Suggestion looks reasonable and I see `beforeAll` usage in our code so it seems to make sense.